.storybook/main.ts

```diff
@@ -38,6 +38,16 @@ const config: StorybookConfig = {
   docs: {
     autodocs: true,
   },
+  webpackFinal: async (config) => {
+    config.resolve = {
+      ...config.resolve,
+    }
+    config.resolve.alias = {
+      ...config.resolve.alias,
+      '@': path.resolve(__dirname, '../src'),
+    }
+    return config
+  },
```

Comment:
`config.resolve = { ...config.resolve }` が何をしているのかパッと見でわからなかったので提案です!

```suggestion
  webpackFinal: async ({ resolve = {}, ...config }) => {
    resolve.alias = {
      ...resolve.alias,
      '@': path.resolve(__dirname, '../src'),
    }
    return {
      ...config,
      resolve,
    }
  },
```

Reply:
単純にconfig.resolveがoptionalで存在しない可能性があるので、存在しているのであればconfig.resolveの設定済みの内容を再設定（undefinedなのであれば空のオブジェクト）とした上でconfig.resolve.aliasを設定している。という感じです！

仮引数で分割代入するのでも良いかなと思いつつ、config自体を分断するよりは実装側で代入する感じの方が読みやすいか？と思って別途commitしてみたので確認してほしいです〜〜🙏

3ea3b4509e0ccade374f1cd04bdee7c852a7d423